dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractFullReadOnlyController.java

```diff
@@ -313,10 +314,9 @@ protected void postProcessResponseEntity( T entity, WebOptions options, Map<Stri
         @PathVariable( "itemId" ) String pvItemId,
         @RequestParam Map<String, String> parameters,
         TranslateParams translateParams,
-        HttpServletResponse response )
+        HttpServletResponse response, @CurrentUser User user )
```

Comment:
Should we use `currentUser` here instead? so we standardise on the same parameter name (even if its not really needed anymore)